ComSemApp/templates/ComSemApp/student/assessment_results.html

```diff
@@ -80,10 +80,97 @@ <h1 class="card-title">Review complete!</h1>
         </div>
     </div>
 </div>
+<!-- Modal -->
+<div class="modal fade" id="confirm-modal" tabindex="-1" role="dialog" aria-labelledby="confirm-modal-label" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="confirm-modal-label">Are you sure?</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <span id="confirm-message"><b>Are you sure you want to request teacher review?</b><br>
+                    Please listen back to your audio recording to ensure that there was a grading or 
+                    transcription error before requesting review.<br>
+                    If you have already done so, continue. 
+                </span>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                <button type="button" class="btn btn-warning" id="confirm-button">Request Review</button>
+            </div>
+        </div>
+    </div>
+</div>
+  
+  
 <div class="btn-group d-flex card-footer" role="group" aria-label="...">
     <a href="{% url 'student:course' course.id %}" class ="btn-success btn w-100 m-1">Return Home</a>
     <a href="{% url 'student:course' course.id %}" class ="btn-success btn w-100 m-1">Try Again</a>
 </div>
+
+<!-- page form holder, input specifies url "speaking_practice_teacher_request" which links to SpeakingPracticeTeacherReviewRequest" -->
+<form id="pageForm" method="POST">
+    <input hidden type="text" id="teacher_review_request" value="{% url 'student:speaking_practice_teacher_request' course.id%}">
+  </form>
+
 <script src="{% static "ComSemApp/js/ComSemRecording-opus.js" %}?{% now "U" %}"></script>
+<script>
+
+// Add click event listener to all request buttons
+$(".request-btn").on("click", function() {
```

Comment:
Replace with arrow functions. Because the body references `this`, you should place `(obj) => {...` and replace all instances of `this` within these bodies with `obj`